channels/cl.m3u

```diff
@@ -184,12 +184,166 @@ http://v3.tustreaming.cl/utvsc/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="",Evavisión Pachanga (Corporación Eva)
+http://159.69.56.148:25461/live/evavision/2r4rfasf/38.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Music",Mundo de la Música
+https://videostreaming.cloudserverlatam.com/8078/8078/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="",Planetatv Kids
+https://mediacpstreamchile.com:1936/8152/8152/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="",Planetatv Music
+https://5eae379fb77bb.streamlock.net:443/planetatvmusic/planetatvmusic/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="",Planetatv Movie
+https://mediacpstreamchile.com:1936/8140/8140/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="",EGM Channel
+https://paneltv.online:1936/8186/8186/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="",LRP Televisión
+https://v2.tustreaming.cl/lrp/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="",U Los Lagos TV
+http://tv.ulagos.cl/web/live.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Sports",Turf Móvil
+https://janus.tvturf.cl/playlist/stream.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Sports",RedBull Batalla de Gallos
+https://videostreaming.cloudserverlatam.com/Batalladegallos/Batalladegallos/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL;HISPAM" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/104654614259430/picture?width=300&height=300" group-title="Religious",Restaurando Vidas Internacional
+http://v4.tustreaming.cl/restaurandovidastv/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Religious",Vida TV
+http://45.161.188.242:88/vidatv/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Religious",Voz de Poder
+https://v2.tustreaming.cl/vozdepoder/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Religious",Unidad Evangelica TV
+https://v2.tustreaming.cl/unidadevangelica/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Religious",TNE
+https://v2.tustreaming.cl/tnetv/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="Religious",Canal ISB (Iglesia San Bernardo)
+https://unlimited1-us.dps.live/isb/isb.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="CL" tvg-language="Spanish" tvg-logo="" group-title="",Spectrum Channel LGBTQ+
+https://vdohd.cl:1936/8078/8078/playlist.m3u8
```

Comment:
I think the server crashed.

Reply:
Why the server crashed? Some streams take a while, but works.

Reply:
It doesn't answer anymore on my side. Was scanning the server and poof, it crashed while doing it.

Reply:
I think of Spectrum Channel is now geoblocked.

Reply:
I don't think so, I think that it's a very unstable server.

Reply:
So, what could you recommend if it will come out as [Timeout] anyway?

Reply:
I must say that in these kind of cases, it's more a server issue than anything. Not 24/7 would be the appropriate tag, even if it's not a channel shutdown.

Reply:
Very fine. I mark as solved and change the tag.